package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.20.0",
+    "lodash.startcase": "^4.4.0",
```

Comment:
I'm using this to convert the data in each category from camelCase to sentence case, e.g. 
`racialDemographicsAtStart` => `Racial Demographics At Start`